CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 - Fix consuming the SDK as pod dependencies for non standard projects not using react-native link [#326](https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/326).
 
+### Misc
+
+- SDK supports react-native v0.57.0.
```

Comment:
Previous version too, since SDK does not require a change I would not document that?